source/includes/wp-api-v3/_product-variations.md

```diff
@@ -624,12 +624,16 @@ woocommerce.get("products/22/variations").parsed_response
 | `parent_exclude` | array   | Limit result set to all items except those of a particular parent ID.                                                                   |
 | `slug`           | string  | Limit result set to products with a specific slug.                                                                                      |
 | `status`         | string  | Limit result set to products assigned a specific status. Options: `any`, `draft`, `pending`, `private` and `publish`. Default is `any`. |
+| `include_status` | string  | Limit result set to product variations with any of the specified statuses. Multiple statuses can be provided as a comma-separated list. Takes precedence over the `status` parameter. Options: `any`, `trash`, `draft`, `pending`, `private`, and `publish`.|
+| `exclude_status` | string  | Exclude product variations from result set with any of the specified statuses. Multiple statuses can be provided as a comma-separated list. Takes precedence over the `include_status` parameter. Options: `trash`, `draft`, `pending`, `private`, and `publish`.|
```

Comment:
Is `future` an option for these params as well?

Reply:
Yes, thanks, updated!